rpcs3/Emu/Audio/AudioBackend.cpp

```diff
@@ -100,13 +100,22 @@ void AudioBackend::normalize(u32 sample_cnt, const f32* src, f32* dst)
 	}
 }
 
-std::pair<AudioChannelCnt, AudioChannelCnt> AudioBackend::get_channel_count_and_downmixer(u32 device_index)
+AudioChannelCnt AudioBackend::get_channel_count(u32 device_index)
```

Comment:
It makes absolutely no sense to split this in two functions.
All you're doing is adding an additional race condition

Reply:
Unified them again